src/EventTickets/Migrations/CreateEventTicketTypesTable.php

```diff
@@ -42,10 +42,10 @@ public function run() {
         $sql = "CREATE TABLE $table (
 			id INT UNSIGNED NOT NULL AUTO_INCREMENT,
 			event_id INT UNSIGNED NOT NULL,
-			label TEXT NULL,
+			title TEXT NULL,
 			description TEXT NULL,
-			price INT UNSIGNED NULL,
-			max_available INT UNSIGNED NULL,
+			price DECIMAL(18,10) NOT NULL,
```

Comment:
This should be reverted to an unsigned integer.

Reply:
In general, currency values should be stored in minor amounts, which is a best practice. We diverge from this in some places, but that is largely for reasons of backwards compatibility.

Reply:
Reverted on 854403bc6894a4baae4d761d07c832a00080f698